apps/central-scan/integration-testing/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK=TRUE
```

Comment:
Allows test to pass without updating images if definition changes only slightly.